src/hooks/useLocationSorting.js

```diff
@@ -0,0 +1,43 @@
+import useSorting from '@folio/stripes-acq-components/lib/AcqList/hooks/useSorting';
+import { useCallback, useEffect } from 'react';
+import { buildSearch, buildSortingObj } from '@folio/stripes-acq-components/lib/AcqList/utils';
+
+export const useLocationSorting = (location, history, resetData, sortableFields, defaultSorting) => {
```

Comment:
what the difference between this hooks and hooks from acq-components?

Reply:
@NikitaSedyx it has deps in useEffect, in acq-components it has empty deps

Reply:
does it make sense to fix in acq-components?

Reply:
Looks like we can pass deps in `acq-components` and pass it, and we don't need it make duplicate of that hook in `bulk-edit`

Reply:
@zburke , @NikitaSedyx  I made changes in acq components https://github.com/folio-org/stripes-acq-components/pull/695 so we don't need our custom `useLocationSorting.js`